projects/packages/backup/src/class-jetpack-backup.php

```diff
@@ -480,6 +524,142 @@ public static function manage_dismissed_backup_review_request( $request ) {
 		return \Jetpack_Options::update_option( 'dismissed_backup_review_' . $request['option_name'], true );
 	}
 
+	/**
+	 * Get site storage size
+	 *
+	 * @return string|WP_Error A JSON object with the site storage size if the request was successful, or a WP_Error otherwise.
+	 */
+	public static function get_site_rewind_size() {
+		$blog_id = \Jetpack_Options::get_option( 'id' );
+
+		$response = Automattic\Jetpack\Connection\Client::wpcom_json_api_request_as_user(
+			'/sites/' . $blog_id . '/rewind/size?force=wpcom',
+			'v2',
+			array(),
+			null,
+			'wpcom'
+		);
+
+		if ( 200 !== wp_remote_retrieve_response_code( $response ) ) {
+			return null;
+		}
+
+		return rest_ensure_response(
+			json_decode( $response['body'], true )
+		);
+	}
+
+	/**
+	 * Get site policies from WPCOM. It includes the storage limit and activity log limit, if apply.
+	 *
+	 * @return string|WP_Error A JSON object with the site storage policies if the request was successful,
+	 *                         or a WP_Error otherwise.
+	 */
+	public static function get_site_rewind_policies() {
+		$blog_id = \Jetpack_Options::get_option( 'id' );
+
+		$response = Automattic\Jetpack\Connection\Client::wpcom_json_api_request_as_user(
+			'/sites/' . $blog_id . '/rewind/policies?force=wpcom',
+			'v2',
+			array(),
+			null,
+			'wpcom'
+		);
+
+		if ( 200 !== wp_remote_retrieve_response_code( $response ) ) {
+			return null;
+		}
+
+		return rest_ensure_response(
+			json_decode( $response['body'], true )
+		);
+	}
+
+	/**
+	 * Get suggested storage addon based on storage usage
+	 *
+	 * @param int $bytes_used      Storage used.
+	 * @param int $bytes_available Storage limit.
+	 * @return string Suggested addon storage slug
+	 */
+	public static function get_storage_addon_upsell_slug( $bytes_used, $bytes_available ) {
+		$bytes_10gb  = 10737418240;
+		$bytes_100gb = 107374182400;
+		$bytes_1tb   = 1099511627776;
```

Comment:
Can we leave a comment here explaining where these IDs are coming from?

Reply:
I think those are not IDs but rather byte sizes. One way to remake this into a more readable form would be:

```php
		$bytes_10gb  = 10 * 1024 * 1024 * 1024;
		$bytes_100gb = 100 * 1024 * 1024 * 1024;
		$bytes_1tb   = 1024 * 1024 * 1024 * 1024;
```

Reply:
You're right 🤦🏻  But still, worth making these more readable.

Reply:
Way better now!